### ISSUE TITLE
Topic in latest ArXiV documents at 2022-03-01 20:24:04

### ISSUE THREAD

**Issue by wanted2** [wanted2/caineng.in#522]
**Is your feature request related to a problem? Please describe.**
Topic in latest ArXiV documents at 2022-03-01 20:24:04
**Describe the solution you'd like**
* Topic 0: 
  * _Patterns of near-crash events in a naturalistic driving dataset:  applying rules mining_
  * _Iterated Belief Change, Computationally_
  * _Formulating Beurling LASSO for Source Separation via Proximal Gradient  Iteration_
  * _Rethinking Immersive Virtual Reality and Empathy_
  * _UnifiedQA-v2: Stronger Generalization via Broader Cross-Format Training_
  * _On change of measure inequalities for $f$-divergences_
  * _Program Synthesis for the OEIS_
  * _Measuring Trustworthiness or Automating Physiognomy? A Comment on Safra,  Chevallier, Grèzes, and Baumard (2020)_
  * _Information Extraction through AI techniques: The KIDs use case at  CONSOB_
  * _Publication and collaboration anomalies in academic papers originating  from a paper mill: evidence from Russia_
* Topic 1: 
  * _Music2Video: Automatic Generation of Music Video with fusion of audio  and text_
  * _Audio Visual Scene-Aware Dialog Generation with Transformer-based Video  Representations_
  * _Reproducibility in Optimization: Theoretical Framework and Limits_
  * _Minimax in Geodesic Metric Spaces: Sion's Theorem and Algorithms_
  * _Long-Time Convergence and Propagation of Chaos for Nonlinear MCMC_
  * _Differentially Private Regression with Unbounded Covariates_
  * _Single Trajectory Nonparametric Learning of Nonlinear Dynamics_
  * _Finding Safe Zones of policies Markov Decision Processes_
  * _Generalized Bayesian Additive Regression Trees Models: Beyond  Conditional Conjugacy_
* Topic 2: 
  * _Graph Spring Network and Informative Anchor Selection for Session-based  Recommendation_
  * _Learning Multi-granularity User Intent Unit for Session-based  Recommendation_
  * _UA-FedRec: Untargeted Attack on Federated News Recommendation_
  * _Hierarchical User Intent Graph Network forMultimedia Recommendation_
  * _Intention Adaptive Graph Neural Network for Category-aware Session-based  Recommendation_
  * _Quality-aware News Recommendation_
  * _Multi-view Intent Disentangle Graph Networks for Bundle Recommendation_
  * _The Unfairness of Active Users and Popularity Bias in Point-of-Interest  Recommendation_
* Topic 3: 
  * _Deep Bayesian ICP Covariance Estimation_
  * _Intelligent Online Selling Point Extraction for E-Commerce  Recommendation_
  * _Active Learning for Point Cloud Semantic Segmentation via  Spatial-Structural Diversity Reasoning_
  * _3D Point Cloud Reconstruction and SLAM as an Input_
  * _PointSCNet: Point Cloud Structure and Correlation Learning Based on  Space Filling Curve-Guided Sampling_
  * _Point Cloud Denoising via Momentum Ascent in Gradient Fields_
  * _Variable Rate Compression for Raw 3D Point Clouds_
  * _On the impact of VR assessment on the Quality of Experience of Highly  Realistic Digital Humans_
  * _PointPCA: Point Cloud Objective Quality Assessment Using PCA-Based  Descriptors_
  * _Reliable Inlier Evaluation for Unsupervised Point Cloud Registration_
* Topic 4: 
  * _Improving Content-Aware Video Streaming in Congested Networks with  In-Network Computing_
  * _Punctuation Restoration_
  * _FoV Privacy-aware VR Streaming_
  * _GenéLive! Generating Rhythm Actions in Love Live!_
  * _DCNGAN: A Deformable Convolutional-Based GAN with QP Adaptation for  Perceptual Quality Enhancement of Compressed Video_
  * _Nebula: Reliable Low-latency Video Transmission for Mobile Cloud Gaming_
  * _MMSys'22 Grand Challenge on AI-based Video Production for Soccer_
  * _Banding vs. Quality: Perceptual Impact and Objective Assessment_
  * _Towards Using Clothes Style Transfer for Scenario-aware Person Video  Generation_
* Topic 5: 
  * _Biased or Not?: The Story of Two Search Engines_
  * _Event-centric Query Suggestion for Online News_
  * _PARM: A Paragraph Aggregation Retrieval Model for Dense  Document-to-Document Retrieval_
  * _Case law retrieval: problems, methods, challenges and evaluations in the  last 20 years_
  * _Literature Review of the Pioneering Approaches in Cloud-based Search  Engines Powered by LETOR Techniques_
  * _Tree-based Search Graph for Approximate Nearest Neighbor Search_
  * _Enabling arbitrary translation objectives with Adaptive Tree Search_
  * _Full-privacy secured search engine empowered by efficient genome-mapping  algorithms_
  * _Query Expansion and Entity Weighting for Query Reformulation Retrieval  in Voice Assistant Systems_
* Topic 6: 
  * _Cross-modal Music Emotion Recognition Using Composite Loss-based  Embeddings_
  * _Video Background Music Generation with Controllable Music Transformer_
  * _Unsupervised Learning of Deep Features for Music Segmentation_
  * _Diversity in the Music Listening Experience: Insights from Focus Group  Interviews_
  * _Enhancing Affective Representations of Music-Induced EEG through  Multimodal Supervision and latent Domain Adaptation_
  * _A Novel Multi-Task Learning Method for Symbolic Music Emotion  Recognition_
  * _S3T: Self-Supervised Pre-training with Swin Transformer for Music  Classification_
  * _Learning long-term music representations via hierarchical contextual  constraints_
  * _Music Sentiment Transfer_
* Topic 7: 
  * _(2.5+1)D Spatio-Temporal Scene Graphs for Video Question Answering_
  * _Anchor Graph Structure Fusion Hashing for Cross-Modal Similarity Search_
  * _An Efficient Pruning Process with Locality Aware Exploration and Dynamic  Graph Editing for Subgraph Matching_
  * _Sign and Basis Invariant Networks for Spectral Graph Representation  Learning_
  * _Spectral embedding and the latent geometry of multipartite networks_
  * _Similarity search on neighbor's graphs with automatic Pareto optimal  performance and minimum expected quality setups based on hyperparameter  optimization_
* Topic 8: 
  * _Automated Reasoning in Non-classical Logics in the TPTP World_
  * _Musical Audio Similarity with Self-supervised Convolutional Neural  Networks_
  * _CAISE: Conversational Agent for Image Search and Editing_
* Topic 9: 
  * _Real-time Over-the-air Adversarial Perturbations for Digital  Communications using Deep Neural Networks_
  * _Reasoning with Scene Graphs for Robot Planning under Partial  Observability_
  * _ReorientBot: Learning Object Reorientation for Specific-Posed Placement_
  * _TwistSLAM: Constrained SLAM in Dynamic Environment_
  * _Coordinate-Aligned Multi-Camera Collaboration for Active Multi-Object  Tracking_
  * _PartAfford: Part-level Affordance Discovery from 3D Objects_
  * _3DRM:Pair-wise relation module for 3D object detection_
  * _DL-SLOT: Dynamic Lidar SLAM and Object Tracking Based On Graph  Optimization_
* Topic 10: 
  * _Domain-Augmented Domain Adaptation_
  * _Transferrable Contrastive Learning for Visual Domain Adaptation_
  * _Multi-Source Unsupervised Domain Adaptation via Pseudo Target Domain_
  * _Attention-based Cross-Layer Domain Alignment for Unsupervised Domain  Adaptation_
  * _Conditional Extreme Value Theory for Open Set Video Domain Adaptation_
  * _BioADAPT-MRC: Adversarial Learning-based Domain Adaptation Improves  Biomedical Machine Reading Comprehension Task_
  * _DGSS : Domain Generalized Semantic Segmentation using Iterative Style  Mining and Latent Representation Alignment_
  * _Few-shot Unsupervised Domain Adaptation with Image-to-class Sparse  Similarity Encoding_
* Topic 11: 
  * _Local Directional Gradient Pattern: A Local Descriptor for Face  Recognition_
  * _R-Theta Local Neighborhood Pattern for Unconstrained Facial Image  Recognition and Retrieval_
  * _Local Quadruple Pattern: A Novel Descriptor for Facial Image Recognition  and Retrieval_
  * _Improving Fake News Detection by Using an Entity-enhanced Framework to  Fuse Diverse Multimodal Clues_
  * _Local Gradient Hexa Pattern: A Descriptor for Face Recognition and  Retrieval_
  * _Centre Symmetric Quadruple Pattern: A Novel Descriptor for Facial Image  Recognition and Retrieval_
* Topic 12: 
  * _Causal discovery for observational sciences using supervised machine  learning_
  * _Learning Latent Causal Dynamics_
  * _Deep Causal Reasoning for Recommendations_
  * _Benign-Overfitting in Conditional Average Treatment Effect Prediction  with Linear Regression_
  * _Diffusion Causal Models for Counterfactual Estimation_
  * _Attacks and Faults Injection in Self-Driving Agents on the Carla  Simulator -- Experience Report_
  * _Exploring with Sticky Mittens: Reinforcement Learning with Expert  Interventions via Option Templates_
  * _Avalanche RL: a Continual Reinforcement Learning Library_
* Topic 13: 
  * _Confidence Calibration for Object Detection and Segmentation_
  * _ciscNet -- A Single-Branch Cell Instance Segmentation and Classification  Network_
  * _SketchEdit: Mask-Free Local Image Manipulation with Partial Sketches_
* Topic 14: 
  * _Multi-Cue Adaptive Emotion Recognition Network_
  * _Information Fusion in Attention Networks Using Adaptive and Multi-level  Factorized Bilinear Pooling for Audio-visual Emotion Recognition_
* Topic 15: 
  * _Clustering by Hill-Climbing: Consistency Results_
  * _Many processors, little time: MCMC for partitions via optimal transport  couplings_
  * _A sampling-based approach for efficient clustering in large datasets_
  * _Using the left Gram matrix to cluster high dimensional data_
  * _Strong Consistency for a Class of Adaptive Clustering Procedures_
  * _Web-Based File Clustering and Indexing for Mindoro State University_
  * _Use Image Clustering to Facilitate Technology Assisted Review_
  * _Optimal Clustering with Bandit Feedback_
  * _Clustering by the Probability Distributions from Extreme Value Theory_
* Topic 16: 
  * _Effect Identification in Cluster Causal Diagrams_
  * _Relational Causal Models with Cycles:Representation and Reasoning_
  * _Adversarial Example Devastation and Detection on Speech Recognition  System by Adding Random Noise_
  * _The Causal Marginal Polytope for Bounding Treatment Effects_
  * _SEC4SR: A Security Analysis Platform for Speaker Recognition_
  * _Interpolation and Regularization for Causal Learning_
  * _Causal Analysis Framework for Recommendation_
  * _Stochastic Causal Programming for Bounding Treatment Effects_
* Topic 17: 
  * _Recent Advances and Challenges in Deep Audio-Visual Correlation Learning_
  * _Perceptual Consistency in Video Segmentation_
* Topic 18: 
  * _Bio-Inspired Audio-Visual Cues Integration for Visual Attention  Prediction_
  * _Sound Adversarial Audio-Visual Navigation_
  * _Visual Acoustic Matching_
  * _Cascaded Multilingual Audio-Visual Learning from Videos_
  * _Implementing simple spectral denoising for environmental audio  recordings_
  * _STEREO: Scientific Text Reuse in Open Access Publications_
* Topic 19: 
  * _Cascaded Asymmetric Local Pattern: A Novel Descriptor for Unconstrained  Facial Image Recognition and Retrieval_
* Topic 20: 
  * _A Robust Multi-Objective Bayesian Optimization Framework Considering  Input Uncertainty_
  * _Robust Multi-Objective Bayesian Optimization Under Input Noise_
  * _Rule-based Evolutionary Bayesian Learning_
* Topic 21: 
* Topic 22: 
* Topic 23: 
  * _Meta-Learning for Simple Regret Minimization_
  * _Non-stationary Bandits and Meta-Learning with a Small Set of Optimal  Arms_
  * _A PDE-Based Analysis of the Symmetric Two-Armed Bernoulli Bandit_
  * _Interacting Contour Stochastic Gradient Langevin Dynamics_
  * _On the influence of roundoff errors on the convergence of the gradient  descent method with low-precision floating-point computation_
  * _MSTGD:A Memory Stochastic sTratified Gradient Descent Method with an  Exponential Convergence Rate_
  * _The Power of Adaptivity in SGD: Self-Tuning Step Sizes with Unbounded  Gradients and Affine Variance_
  * _Tackling benign nonconvexity with smoothing and stochastic gradients_
* Topic 24: 
  * _Provable Stochastic Optimization for Global Contrastive Learning: Small  Batch Does Not Harm Performance_
* Topic 25: 
  * _Improving Robustness of Convolutional Neural Networks Using Element-Wise  Activation Scaling_
  * _PQuAD: A Persian Question Answering Dataset_
  * _When do GANs replicate? On the choice of dataset size_
  * _Robust Training under Label Noise by Over-parameterization_
* Topic 26: 
  * _A Survey on Patients Privacy Protection with Stganography and Visual  Encryption_
  * _FakeAVCeleb: A Novel Audio-Video Multimodal Deepfake Dataset_
* Topic 27: 
  * _Large-scale Stochastic Optimization of NDCG Surrogates for Deep Learning  with Provable Convergence_
* Topic 28: 
  * _FNR: A Similarity and Transformer-Based Approach to Detect Multi-Modal  Fake News in Social Media_
  * _Efficient and Differentiable Conformal Prediction with General Function  Classes_
* Topic 29: 
  * _Deconstructing Distributions: A Pointwise Framework of Learning_
* Topic 30: 
* Topic 31: 
  * _A Real-World Implementation of Unbiased Lift-based Bidding System_
  * _Bidding Agent Design in the LinkedIn Ad Marketplace_
* Topic 32: 
  * _Deepfake Detection for Facial Images with Facemasks_
  * _Robotic Telekinesis: Learning a Robotic Hand Imitator by Watching Humans  on Youtube_
* Topic 33: 
  * _FUNQUE: Fusion of Unified Quality Evaluators_
  * _Perceptual Evaluation of 360 Audiovisual Quality and Machine Learning  Predictions_
* Topic 34: 
  * _Large-scale Personalized Video Game Recommendation via Social-aware  Contextualized Graph Neural Network_
  * _Learning to Identify Perceptual Bugs in 3D Video Games_
  * _How player and opponent personalities influence cooperative gameplay_
  * _Player Dominance Adjustment in Games_
  * _Fighting Game Commentator with Pitch and Loudness Adjustment Utilizing  Highlight Cues_
  * _Integrating psychotherapy practices and gamified elements in novel game  mechanics for stress relief_
* Topic 35: 
  * _Multiscale Crowd Counting and Localization By Multitask Point  Supervision_
  * _Reinforcing Local Feature Representation for Weakly-Supervised Dense  Crowd Counting_
  * _Wisdom of (Binned) Crowds: A Bayesian Stratification Paradigm for Crowd  Counting_
  * _FusionCount: Efficient Crowd Counting via Multiscale Feature Fusion_
  * _An End-to-End Transformer Model for Crowd Localization_
* Topic 36: 
* Topic 37: 
  * _Towards Reducing Manual Workload in Technology-Assisted Reviews:  Estimating Ranking Performance_
* Topic 38: 
  * _Extraction of Product Specifications from the Web -- Going Beyond Tables  and Lists_
  * _MAVE: A Product Dataset for Multi-source Attribute Value Extraction_
* Topic 39: 
* Topic 40: 
* Topic 41: 
  * _Unwinding Rotations Improves User Comfort with Immersive Telepresence  Robots_
* Topic 42: 
  * _Data Overlap: A Prerequisite For Disentanglement_
* Topic 43: 
* Topic 44: 
  * _A Complete Criterion for Value of Information in Soluble Influence  Diagrams_
  * _Differential privacy for symmetric log-concave mechanisms_
* Topic 45: 
  * _Multi-Objective Dual Simplex-Mesh Based Deformable Image Registration  for 3D Medical Images -- Proof of Concept_
* Topic 46: 
  * _Nonlinear Discrete Optimisation of Reversible Steganographic Coding_
* Topic 47: 
  * _Parameterized Intractability for Multi-Winner Election under the  Chamberlin-Courant Rule and the Monroe Rule_
* Topic 48: 
  * _On PAC-Bayesian reconstruction guarantees for VAEs_
* Topic 49: 
  * _Click-Through Rate Prediction in Online Advertising: A Literature Review_

**Describe alternatives you've considered**
None
**Additional context**
None

**Comment by wanted2** [comment]
This issue was created by an aggregate batch
